mono/tests/test-runner.cs

```diff
@@ -96,6 +97,13 @@ class TestInfo {
 					}
 					runtime = args [i + 1];
```

Comment:
while reviewing I noticed the `--runtime` option is duplicated below, can you please remove the other one?